x11-misc/nitrogen/nitrogen-1.6.1-r2.ebuild

```
@@ -27,8 +27,13 @@ DEPEND="
 	xinerama? ( x11-base/xorg-proto )
 "
 
+PATCHES=(
+	"${FILESDIR}/${P}-fix-appdata-install-location.patch"
```

It is always a good idea to leave a comment to the upstream submission of the patch:

```suggestion
	"${FILESDIR}/${P}-fix-appdata-install-location.patch" # https://github.com/l3ib/nitrogen/pull/156
```